doc/lua_api.txt

```diff
@@ -2590,6 +2591,37 @@ An `InvRef` is a reference to an inventory.
 * `get_location()`: returns a location compatible to `minetest.get_inventory(location)`
     * returns `{type="undefined"}` in case location is not known
 
+### `AreaStore`
+A fast access data structure to store areas, and find areas near a given position or area.
+Every area has a `data` string attribute to store additional information.
+You can create an empty `AreaStore` by calling `AreaStore()`, or `AreaStore(type_name)`.
+If you chose the parameter-less constructor, a fast implementation will be automatically chosen for you.
+#### Methods
+* `get_area(id, borders, data)`: returns the area with the id `id`. (optional) Boolean values `borders` and `data` control what's copied.
+* `get_areas_for_pos(pos, borders, data)`: returns all areas that contain the position `pos`. (optional) Boolean values `borders` and `data` control what's copied.
+* `get_areas_in_area(area, borders, data)`: returns all areas that contain all nodes inside `area`. If `accept_overlap` is true, also areas are returned that have nodes in common with `area`. (optional) Boolean values `borders` and `data` control what's copied.
+    * `area`:
+      {
+        min = position,
+        max = position,
+      }
+* `insert_area(area)`: inserts an area into the store. Returns the id if successful, false otherwise.
+    * `area`: Positions min and max describe edges of the area, they don't have to be minimal and maximal values, the implementation will automatically swap values accordingly.
+      {
+        min = position,
+        max = position,
+        data = string,
+      }
+* `reserve(count)`: reserves resources for at most `count` many contained areas. Only needed for efficiency, and only some implementations profit.
+* `remove_area(id)`: removes the area with the given id from the store, returns success.
+* `set_cache_params(params)`: sets params for the included prefiltering cache. Calling invalidates the cache, so that its elements have to be newly generated.
+    * `params`:
+      {
+        enabled = boolean, -- whether to enable, default true
+        block_radius = number, -- the radius of the blocks the cache generates prefiltered lists for, minimum 16, default 64
```

Comment:
Perhaps.  You could just do this though: `- the radius (in nodes) of the area the cache generates prefiltered lists for...`  Note `blocks`->`area`.